src/Exporter/Plugin/OrderResourcePlugin.php

```diff
@@ -192,4 +219,10 @@ private function getOrder(OrderItemInterface $resource)
     {
         return $resource->getOrder();
     }
+
+    private function isExist($class, $proxyObject)
+    {
+        $this->entityManager->detach($proxyObject);
+        return $this->entityManager->find($class, $proxyObject->getId()) != null;
```

Comment:
Performance issue is here because of **find**. You will run a single query to get all entities you use.